packages/api/package.json

```diff
@@ -16,7 +16,7 @@
     "db-types": "./scripts/cli.js db-types"
   },
   "author": "Hugo Dias <hugomrdias@gmail.com> (hugodias.me)",
-  "license": "MIT",
+  "license": "(Apache-2.0 AND MIT)",
```

Comment:
@vasco-santos actually one last question ( sorry for being such a PITA!!! )

Is this field supposed to be a SPDX identifier? If yes it should read ( like what you just copied ) `Apache-2.0 OR MIT`

Reply:
Yes, you are right! https://docs.npmjs.com/cli/v8/configuring-npm/package-json#license

We have AND in all the repos in nft and web3, but it should be as you suggest